### PR TITLE
Curated_85_batch

### PR DESCRIPTION
**Pull request by umasrisankarlal**
Complete list of batch 85 with corrected details

### CODE REVIEW

**Comment by TanskaAnnna** [comment]
@umasrisankarlal  your Validate PR have failed. This is because you have changes in more than 1 file. This PR ncludes changes in indication_paths.json as well as indication_paths.yaml. You may need to delete your fork to avoid that.

**Comment by umasrisankarlal** [comment]
Hello, @TanskaAnnna I was not able to delete my fork as it threw an error. Seems now PR has no conflicts. Can you please confirm, if I need to delete my fork and create it again?

**Comment by TanskaAnnna** [comment]
@umasrisankarlal there are still conflicts. Can you share a screenshot for your error please?

**Comment by TanskaAnnna** [comment]
Ok it's fixed now.

File: submission.yaml [hunk 1]
```diff
@@ -1 +1,1143 @@
...
+    -   id: GO:0006939
+        label: BiologicalProcess
+        name: Smooth muscle contraction
+    -   id: MESH:D029424
+        label: Disease
+        name: Chronic obstructive lung disease
+    reference:
+    - https://go.drugbank.com/drugs/DB12086
+    - https://drugs.ncats.io/drug/8G15T83E6I
+    comment: Oxitropium is used as Oxitropium bromide for treatment
```
Comment (TanskaAnnna) [inline submission.yaml:282]: @umasrisankarlal could you remove that space in line 282 please?

Comment (umasrisankarlal) [reply submission.yaml]: @TanskaAnnna , corrected the changes, but the PR shows failed.
Can you please check and suggest?

Comment (TanskaAnnna) [reply submission.yaml]: @umasrisankarlal it failed because you have changes in more than one file. Only changes to the submission.yaml are allowed.

Comment (umasrisankarlal) [reply submission.yaml]: Hello @TanskaAnnna , I deleted my fork and created a new one. New one is updated with recent changes

**Comment by TanskaAnnna** [comment]
2 files changed, closing this PR

**Pull request closed without merging**